sdk/lib/rtl/heap.c

```diff
@@ -1100,11 +1124,10 @@ RtlpInitializeHeapSegment(IN OUT PHEAP Heap,
             FreeEntry->SegmentOffset = SegmentIndex;
             FreeEntry->Flags = HEAP_ENTRY_LAST_ENTRY;
             FreeEntry->Size = (SegmentCommit >> HEAP_ENTRY_SHIFT) - Segment->Entry.Size;
-        }
 
-        /* Register the Free Heap Entry */
-        if (FreeEntry)
+            /* Register the Free Heap Entry */
             RtlpInsertFreeBlock(Heap, (PHEAP_FREE_ENTRY)FreeEntry, FreeEntry->Size);
```

Comment:
Ignoring the new return value here is OK?

Reply:
Yes, most places don't need to do anything with it (which is why it was void before).